code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -128,6 +140,7 @@
 /datum/eldritch_knowledge/curse
 	var/timer = 5 MINUTES
 	var/list/fingerprints = list()
+	var/list/dna = list()
```

Comment:
This `dna` list is not used. Its usage isn't implemented in tg either. So you have a choice. You implement it, or you don't port the remnants of dna check.

Reply:
I think this is a leftover of some sorts.